corehq/apps/export/templates/export/dialogs/delete_custom_export_dialog.html

```diff
@@ -1,10 +1,15 @@
+{% load hq_shared_tags %}
 {% load i18n %}
 <div id="delete-export-modal-{{ export.get_id }}" class="modal hide fade">
     <div class="modal-header">
         <a class="close" data-dismiss="modal">×</a>
         <h3>{% trans "Delete Export for" %} "{{ export.name }}"?</h3>
     </div>
+    {% if request|toggle_enabled:"NEW_EXPORTS" %}
+    <form name="drop_report" action="{% url "delete_new_custom_export" domain export.type export.get_id %}" method="post">
```

Comment:
If the toggle is enabled will ALL exports on the page be new exports? Or is that still in the pipeline?